packages/core/src/artifacts.ts

```diff
@@ -78,6 +78,10 @@ export function getBuiltKeystoneConfigurationPath(cwd: string) {
   return path.join(cwd, '.keystone/config.js');
 }
 
+function posixify(s: string) {
+  return s.split(path.sep).join('/');
```

Comment:
This might be OK for Windows, as `/` is not allowed in Windows filenames, but for Unix the `split` would be wrong, but probably still OK as it's immediately rejoined with `/`...